docs/source/examples/classification_and_regression.rst

```diff
@@ -0,0 +1,296 @@
+    valid_dataset = datasets.CelebA(data_root, split="valid", target_type=["attr", "landmarks"], transform=transforms)
+    test_dataset = datasets.CelebA(data_root, split="test", target_type=["attr", "landmarks"], transform=transforms)
+    
+Here we can see how each dataset sample looks like:
+
+.. code-block:: python
+
+    print(train_dataset[0])
+
+.. image:: /_static/img/classification_and_regression/out.png
+
+Regarding the complexity of the problem and the number of training/valid samples, we have a huge number of training/validation images. Since there are not a considerable variation between images (e.g., the eye coordinates in images do not vary considerably), using all images in the dataset is not necessary and will only increase the training time. Hence, we can seperate and use a portion of data as below:
+
+.. code-block:: python
+
+    train_subset = Subset(train_dataset, np.arange(1, 50000))
+    valid_subset = Subset(valid_dataset, np.arange(1, 2000))
+    test_subset = Subset(test_dataset, np.arange(1, 1000))
+    train_dataloader = DataLoader(train_subset, batch_size=batch_size, shuffle=True)
+    valid_dataloader = DataLoader(valid_subset, batch_size=batch_size, shuffle=False)
+    test_dataloader = DataLoader(test_subset, batch_size=batch_size, shuffle=False)
+
+Here, we can see an example from the training dataset. It shows an image of a person, printing the gender and also showing the location of the eyes. It is worth mentioning that as we resize the image, the coordinates of the eyes should also be changed with same ratio.
+
+.. code-block:: python
+
+    sample_number = 189
+    image = train_dataset[sample_number][0]
+    image = image.permute(1, 2, 0).detach().numpy()
+    image_rgb = cv2.cvtColor(np.float32(image), cv2.COLOR_BGR2RGB)
+    image_rgb = image_rgb * imagenet_std + imagenet_mean
+    gender = "male" if int(train_dataset[sample_number][1][0][gender_index]) == 1 else "female"
+    print("Gender is:", gender)
+    w, h = 218, 178
+    (x_L, y_L) = train_dataset[sample_number][1][1][
+        0:2
+    ]  # The coordinates vector of the datasets starts with X_L, y_L, X_R, y_R
+    (x_R, y_R) = train_dataset[sample_number][1][1][2:4]
+    w_scale = image_size / w
+    h_scale = image_size / h
+    x_L, x_R = (x_L * h_scale), (x_R * h_scale)  # rescaling for the size of (224,224) and finaly to the range of [0,1]
+    y_L, y_R = (y_L * w_scale), (y_R * w_scale)
+    x_L, x_R = int(x_L), int(x_R)
+    y_L, y_R = int(y_L), int(y_R)
+    image_rgb = cv2.drawMarker(image_rgb, (x_L, y_L), (0, 255, 0))
+    image_rgb = cv2.drawMarker(image_rgb, (x_R, y_R), (0, 255, 0))
+    image_rgb = cv2.cvtColor(np.float32(image_rgb), cv2.COLOR_BGR2RGB)
+    image_rgb = np.clip(image_rgb, 0, 1)
+    plt.imshow(image_rgb)
+    plt.axis("off")
+    plt.show()
+
+.. image:: /_static/img/classification_and_regression/dataset_sample.png
```

Comment:
The image here does not fit what appears in the notebook, even if we use the same sample number.